app/src/components/main/tabsSwitcher/tabsSwitcher.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
```

Comment:
We already have components/buttons/toggleButton common component, so there s no need in implementation of another one.

Reply:
Fixed